.github/CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+# This is a comment.
```

Comment:
This is more a query than any recommendation
Should we keep all this default commentary from the code owner file? Not that it is hurting anything; but still.  

```
# https://help.github.com/en/articles/about-code-owners

# These owners will be the default owners for everything in
# the repo. Unless a later match takes precedence,
*       @atbagga @gauravsaralMs @ishitam8

# These owners own any file in the `/docs`
# directory in the root of your repository.
/doc/  @geverghe*/```